packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -505,12 +529,16 @@ class _DefaultPub implements Pub {
     return values.join(':');
   }
 
-  String? _getRootPubCacheIfAvailable() {
+  /// There is 3 ways to get the cache path
```

Comment:
Same notes about dartdocs should start with a high level summary: https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary